js/main.js

```diff
@@ -23,5 +27,41 @@ function addEntry(event) {
   $form.reset();
```

Comment:
You should be adding the newly submitted entry to the list of entries here, too. You are currently only making DOM trees for new elements when the page loads, but you also need to generate one for each new entry and add it to the DOM.